src/ripple/rpc/handlers/GetCounts.cpp

```diff
@@ -121,21 +113,21 @@ Json::Value doGetCounts (RPC::Context& context)
     textTime (uptime, s, "second", 1s);
     ret[jss::uptime] = uptime;
```

Comment:
Also not in your changeset, but I'd put this into it's own block. Something like:
```
    {
        using namespace std::chrono_literals;
        std::string uptime;
        uptime.reserve(16);
        auto s = UptimeClock::now();
        textTime(uptime, s, "year", 365 * 24h);
        textTime(uptime, s, "day", 24h);
        textTime(uptime, s, "hour", 1h);
        textTime(uptime, s, "minute", 1min);
        textTime(uptime, s, "second", 1s);
        ret[jss::uptime] = uptime;
    }
```

Reply:
By the way, @HowardHinnant: this kind of thing (format duration to human-readable form) would probably make a nice addition to your time library.

Reply:
Just FYI - a "year" is not a static time span of 365 days. Uptime in that regions is unhealthy anyways, so I'd recommend leaving it out.

Reply:
Howard is the master of all things `<chrono>` so I'm sure he can sort it out ;)

Reply:
I agree with Markus.  The year unit is both never used and somewhat ambiguous if it ever were to be used.  Simply removing that one line of code leaves a correct and accurate uptime.

Whether it should be done as part of this PR, or in a separate PR is a another question.

Reply:
It can be removed if that's the consensus. I didn't change the core of this code, just refactored it so I could re-use it in the `OverlayImpl` class.